analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ConsumeValueTaskCorrectly.CSharp6.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Threading.Tasks;
```

Comment:
This can be `ConsumeValueTaskCorrectly.NetCore.cs`
Like this [file here](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TaskConfigureAwait.NetCore.cs).